cpu/esp32/periph/spi.c

```diff
@@ -312,6 +312,8 @@ void IRAM_ATTR spi_release(spi_t bus)
 static const char* _spi_names[] = { "CSPI/FSPI", "HSPI", "VSPI"  };
 #elif defined(CPU_FAM_ESP32C3)
 static const char* _spi_names[] = { "SPI", "FSPI"  };
+#elif defined(CPU_FAM_ESP32S2)
+static const char* _spi_names[] = { "SPI", "FSPI", "HSPI" };
```

Comment:
Do the FSPI/HSPI names still make sense if they are only used for legacy reasons?

Doesn't have to be here, but would a follow up where we rename FSPI -> SPI2 and HSPI -> SPI2 make sense?